components/ILIAS/Forum/PRIVACY.md

```diff
@@ -0,0 +1,101 @@
+
+**Moderator Functions / Approve Postings** (default: off) enables posts in the forum to be published only after they have been read and approved by moderators.
+
+**Privacy / Enable Statistics** (default: off) enables all forum users to see who has published how many posts.
+
+**Privacy / Posting with pseudonym** (default setting: off) allows users to publish posts in the forum anonymously or using a pseudonym. Forum posts that have already been anonymized remain anonymous even after a changeover.
+
+### Notifications
+
+For forums within courses and groups, you can set here whether users of the forum automatically receive notifications by e-mail, whether they have to activate it manually or whether they can deactivate it individually (default setting: manual activation).
+
+### News settings
+
+If RSS is activated in ILIAS under “Administration > News and Web Feeds”, you can set here whether an RSS feed is offered for the forum. The “Public Notification” setting enables this feed to contain forum posts without authentication (default setting: off).
+
+## Data being stored
+
+The **ID of the creating user** and the time of creation are saved for the entire forum. If the forum settings are changed, the **ID of the user making the change** and the time of the change are saved.
+
+The **title, description** and a manually designed **start page** of the forum may contain personal information provided by the creator.
+
+The **titles and texts of the posts** and topics in the forum are created by the forum participants themselves and may contain personal information, e.g. the names of other forum participants. Posts may also contain images and file attachments.
+
+With every post or draft of a post, the **ID** of the user who created it and the **time** of creation and publication are saved. When changes are made (including censorship or re-release), the time and ID of the user making the change are saved.
+
+If anonymization is set in the forum, the ID of the user and the **pseudonym** entered are saved internally with a post.
+
+The **moderator** function of a user is saved via the ILIAS access control system by the moderator role they belong to. For **censored posts**, the time and comment are saved and are linked to the ID of the censor.
+
+**Notification settings** for the forum or individual topics are saved for users (notification of new topics, modification or censorship of posts, deletion of posts or topics). The settings are linked to the **IDs** of the user who made them and to whom they apply.
+
+The **read status** of posts and the **sorting** of topics is saved for individual users and used to customize the display.
+
+If forum topics can be given a 5-star rating, **individual ratings** are saved by the rating component of ILIAS.
+
+## Data being presented
+
+The title, description, start page of the forum as well as the titles of the topics and the posts may contain **manually created personal information**.
```

Comment:
Change to:
"The title, description, and start page or the forum, as well as, ...".